src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/DeleteDataFilePlanViewModel.cs

```diff
@@ -0,0 +1,23 @@
+#nullable enable
+using System.Collections.Generic;
+using System;
+using Newtonsoft.Json;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+
+public record DeleteDataFilePlanViewModel
+{
+    [JsonIgnore]
+    public Guid ReleaseId { get; init; }
+
+    [JsonIgnore]
+    public Guid SubjectId { get; init; }
+
+    public DeleteDataBlockPlanViewModel DeleteDataBlockPlan { get; init; } = null!;
+
+    public List<Guid> FootnoteIds { get; init; } = null!;
+
+    public DeleteApiDataSetVersionPlanViewModel? LinkedApiDataSetVersionDeletionPlan { get; init; }
```

Comment:
As per other PR - this could be simplified to `DeleteApiDataSetVersionPlan`

Reply:
Done :)